pkg/preflight/execute.go

```diff
@@ -0,0 +1,81 @@
+package preflight
+
+import (
+	"encoding/json"
+	"time"
+
+	"github.com/pkg/errors"
+	"github.com/replicatedhq/kotsadm/pkg/logger"
+	"github.com/replicatedhq/kotsadm/pkg/persistence"
+	troubleshootv1beta1 "github.com/replicatedhq/troubleshoot/pkg/apis/troubleshoot/v1beta1"
+	troubleshootpreflight "github.com/replicatedhq/troubleshoot/pkg/preflight"
+	"go.uber.org/zap"
+	"k8s.io/client-go/rest"
+)
+
+// execute will execute the preflights using spec in renderedPreflights.
+// This spec should be rendered, no template functions remaining
```

Comment:
this comment feels wrong, since the func is passed an object

Reply:
It's correct.  the var name is wrong, but it's a rendered and unmarshalled object, not a string.

updating var name in the comment